devtools/client/framework/toolbox.js

```diff
@@ -45,6 +45,14 @@ const { BrowserLoader } = ChromeUtils.import(
   "resource://devtools/client/shared/browser-loader.js"
 );
 
+// [Replay] - Import getRecordingState to obscure dev tools while recording
+const { getRecordingState } = ChromeUtils.import(
+  "resource://devtools/server/actors/replay/connection.js"
+);
+const { pingTelemetry } = ChromeUtils.import(
+  "resource://devtools/server/actors/replay/telemetry.js"
+);
+
```

Comment:
Nice - i wonder if you're already in here, if it is possible to prevent the devtools client / server from even starting at all?

Reply:
I think we'd still want to message the user that they tried to do something that is currently not allowed lest they think devtools is just broken. I'm sure I could hijack it further but if the goal is to eventually record devtools too, I think this is probably sufficient. WDYT?

Reply:
For sure, we definitely want to show the label. My concern was that initializing devtools might also trigger some issues and might not look as nice, but this is a great first step and definitely sufficient for landing